modules/admin-portal/pages/monitoring/queries.adoc

```diff
@@ -0,0 +1,22 @@
+= Queries
+
+The Query monitoring page gives detailed information about the queries running on a cluster and their performance.
+
+== Overview
+
+image::query-monitoring-overview.png[]
+
+In the Overview tab, you can view currently running queries broken down by node and graph.
+In the Action column for each query, there are buttons to view the individual query history and to stop the query entirely.
+
+You can also see Queries Per Second (QPS), timeout, and latency.
+
+At any point, click the Refresh icon in the upper right corner of the page to refresh the view with the latest statistics.
+
+== Current Query Performance
+
+The Current Query Performance tab shows the same information as the Current Running Queries section on the Overview page.
```

Comment:
Overview page only show 5 records, but current query performance tab shows all the running queries.